en/identity-server/6.1.0/docs/deploy/configure-a-read-write-active-directory-user-store.md

```diff
@@ -113,6 +113,7 @@ since SCIM is enabled by default from the WSO2 Identity Server 5.10.0 onwards.
 
 !!! note
     It is required to edit the claim mappings in WSO2 IS according to the user claims of the Active Directory version you have configured.<br />
+    Additioanlly the `Username` and `User ID` claims need to be mapped correctly to the values used for `Username Attribute` and `User ID Attribute` properties in the userstore configuration for authentication to work properly.<br /><br />
     Before starting the server, edit the `<IS_HOME>/repository/conf/claim-config.xml` configuration file and change the `AttributeID` of the `Created Time` and `Last Modified Time` claims to `whenCreated` and `whenChanged` respectively.
```

Comment:
Shall we include we need to add username/userid mapping to claim-config.xml file as well? The current version does not implies it is a requirement.

Reply:
The change explains username/userid is a requirement
The link to mapping documentation [1] explains both scenarios (through UI and claim-config.xml)
Additionally claim-config.xml change is only applicable before the server startup as documented in [1].
Since this is well explained in [1], do we need to repeat in these changes as well ?

[1] https://is.docs.wso2.com/en/6.1.0/guides/dialects/edit-claim-mapping/